Lib/http/server.py

```diff
@@ -1294,6 +1292,9 @@ def test(HandlerClass=BaseHTTPRequestHandler,
                         default='HTTP/1.0',
                         help='conform to this HTTP version '
                              '(default: %(default)s)')
+    parser.add_argument('-c', '--content-type',  # parsed into content_type
```

Comment:
This feature is not so often used to spent a letter for short option. `'--content-type'` should be enough. Maybe `'--default-content-type'`.

Reply:
Fair enough.